### PR TITLE
Change UI for enableing user space instrumentation.

### PR DESCRIPTION
**Pull request by danielfenner**
Previously this was a checkbox "Enable user space instrumention".
Now it is a combo box labeled "Dynamic instrumentation method" with the two
methods "Kernel (uprobes)" and "Orbit". The tooltip reads:
"Method 'Orbit' has lower overhead but is currently still experimental."

The combo box is only visible in devmode.

### CODE REVIEW

File: src/OrbitQt/CaptureOptionsDialog.ui [hunk 1]
```diff
@@ -9,8 +9,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>478</width>
-    <height>553</height>
+    <width>609</width>
```
Comment (dpallotti) [inline src/OrbitQt/CaptureOptionsDialog.ui:12]: Why the increase in width?

File: src/OrbitQt/CaptureOptionsDialog.ui [hunk 2]
```diff
@@ -166,20 +166,71 @@
...
+           <property name="rightMargin">
+            <number>0</number>
+           </property>
+           <property name="bottomMargin">
+            <number>0</number>
+           </property>
+           <item>
+            <widget class="QLabel" name="label_2">
+             <property name="text">
+              <string>Dynamic Instrumentation Method:</string>
```
Comment (dpallotti) [inline src/OrbitQt/CaptureOptionsDialog.ui:196]:
```suggestion
              <string>Dynamic instrumentation method:</string>
```
Compare with the other options, that are not capitalized. E.g., "Callstack unwinding method".

Comment (danielfenner) [reply src/OrbitQt/CaptureOptionsDialog.ui]: done

File: src/OrbitQt/CaptureOptionsDialog.ui (hunk 1 above)
Comment (danielfenner) [reply src/OrbitQt/CaptureOptionsDialog.ui]: I don't know; this dropped out of the ui designer. I changed it back. Surprisingly it does not makle the slightest differrence what number I put there. The dialog always has identical dimensions.

**Review by dpallotti — APPROVED** [review]

**Pull request merged**